binaries/oli/src/config/mod.rs

```diff
@@ -0,0 +1,162 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use std::collections::HashMap;
+use std::fs;
+use std::path::Path;
+use std::str::FromStr;
+
+use anyhow::{anyhow, Result};
+use opendal::{services, Operator, Scheme};
+use serde::Deserialize;
+use toml;
+
+type StringMap<T> = HashMap<String, T>;
```

Comment:
I prefer remove this type, it's over designed.

Reply:
The purpose of this type is to save me some typings as well, I previously need to reference this type multiple times. Now that this type is only referenced once, I could remove this type.

Reply:
Done